### DISCUSSION TITLE
对lamp-web-pro进行打包问题

### DISCUSSION THREAD

**Discussion by HelloJasonZ** [zuihou/zuihou-admin-cloud#325] [02.提问]
在对前端项目进行build时候，执行 pnpm run build:prod:none，最终打包失败。出现了：build.rollupOptions.external
![image](https://github.com/dromara/lamp-cloud/assets/20830000/70ca782e-4368-4d43-8ba0-01d533ce5746)

**Comment by zuihou** [comment]
没有， 先pnpm install , 然后在 pnpm run build:prod:none

**Comment by zuihou** [comment]
<img width="415" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/6d324797-b71f-40af-bd69-b78e73e6ba4e">

**Reply by HelloJasonZ** [reply]
![image](https://github.com/dromara/lamp-cloud/assets/20830000/60e9ba96-af50-4ff0-ad6d-cd99a2aa210a)
版本是这样，进了pnpm install , 然后在 pnpm run build:prod:none，还是一样的错误信息
